imports/plugins/core/orders/client/templates/workflow/shippingInvoice.html

```diff
@@ -94,6 +94,9 @@
     <hr>
 
     <div class="flex">
+      {{#if orderCancelled }}
+          <button class="btn btn-danger  flex-item-fill" type="button">Cancelled Order</button>
```

Comment:
How many indentations are we using on the view? Also, there seems to be much space between `danger` and `Flex`